core/tests/unittest.py

```
@@ -292,23 +292,30 @@ def test_cases(m):
+        if result.skippedNum > 0:
+            result_strs.append(f"{result.skippedNum} skipped")
+        if result.failuresNum > 0:
+            result_strs.append(f"{result.failuresNum} failed")
+        if result.errorsNum > 0:
+            result_strs.append(f"{result.errorsNum} errored")
+        if result_strs:
+            msg += " (" + ", ".join(result_strs) + ")"
+        print(msg)
+        wasSuccessful = result.wasSuccessful()
+    except Exception:
+        pass
```

Ruff (0.14.14)

`try`-`except`-`pass` detected, consider logging the exception

(S110)